### DISCUSSION TITLE
求助：使用 next 主题后，github actions 运行报错，不知道怎么解决

### DISCUSSION THREAD

**Discussion by xWenChen** [hugo-next/hugo-theme-next#122] [Q&A]
hugo 搭建个人博客，使用 next 主题。github actions 运行报错，不知道怎么解决。求指点

![hugo next 错误提示](https://github.com/hugo-next/hugo-theme-next/assets/36958243/8352e675-cc6b-472f-b163-ee042e3465d3)

**Comment by elkan1788** [comment]
看起来像是你没有配置 `CDN` 的使用方式，请确认下本地是否可以正常运行，并提供详细的配置文件。

**Comment by xWenChen** [comment]
配置文件弄错了，我自己解了
